### PR TITLE
Feature/80 pdf download

### PR DESCRIPTION
**Pull request by megheaiulian**
Downloads images in a PDF file.

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -14,7 +14,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@master
+        uses: actions/checkout@v2
+
+      - run: git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
```
Comment (cristinecula) [inline .github/workflows/workflow.yml]: @megheaiulian if you put in @v1, it works:
https://github.com/Neovici/cosmoz-frontend/issues/1793

Comment (megheaiulian) [reply .github/workflows/workflow.yml]: Yeah but v2 allows a bit more control over what we fetch during checkout. e.g: fetch only master and the pr branch.

Comment (cristinecula) [reply .github/workflows/workflow.yml]: Isn't that what v1 does?

Comment (megheaiulian) [reply .github/workflows/workflow.yml]: Went with v1

File: cosmoz-image-viewer.js [hunk 2]
```diff
@@ -653,7 +613,7 @@ class CosmozImageViewer extends translatable(mixinBehaviors([
 	_shouldLoad(currentImageIndex, index) {
 		/*
 		 * Workaround for https://github.com/Neovici/cosmoz-image-viewer/issues/21
-		 * Changing the images array updates the `data-src` attribute on the items,
+		 * Changing the imagesdownload(details, this.credentials)) array updates the `data-src` attribute on the items,
```
Comment (cristinecula) [inline cosmoz-image-viewer.js]: I think you pasted in error here

File: package.json [hunk 3]
```diff
@@ -67,28 +66,29 @@
...
-		"eslint": "^5.0.0",
-		"eslint-plugin-html": "^5.0.0",
+		"eslint": "^6.0.0",
+		"eslint-plugin-html": "^6.0.0",
 		"eslint-plugin-import": "^2.17.3",
-		"eslint-plugin-mocha": "^5.0.0",
-		"husky": "^3.0.9",
+		"eslint-plugin-mocha": "^6.0.0",
+		"husky": "^4.0.0",
+		"jspdf": "^1.5.3",
```
Comment (cristinecula) [inline package.json]: this needs to be in "dependencies", since it's not tooling.

**Comment by megheaiulian** [comment]
@cristinecula Fixes the very stupid mistakes :). Please review again.

**Pull request merged**